pmm-app/tests/DbaaS/verifyDBaaS_test.js

```diff
@@ -0,0 +1,242 @@
+    I.refreshPage();
+    dbaasPage.checkCluster(clusterName2, true);
+    dbaasPage.checkCluster(clusterName1, true);
+  });
+
+Scenario('Verify DB Cluster Tab Page Elements & Steps Background @dbaas @not-pr-pipeline',
+  async (I, dbaasPage, dbaasAPI) => {
+    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
+      await dbaasAPI.apiRegisterCluster(process.env.kubeconfig_minikube, clusterName);
+    }
+
+    I.amOnPage(dbaasPage.url);
+    dbaasPage.checkCluster(clusterName, false);
+    I.click(dbaasPage.tabs.dbClusterTab.dbClusterTab);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.addDbClusterButton, 30);
+    I.click(dbaasPage.tabs.dbClusterTab.addDbClusterButton);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.monitoringWarningLocator, 30);
+    I.waitForText(dbaasPage.monitoringWarningMessage, 30);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.basicOptions.fields.clusterNameField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.basicOptions.fields.kubernetesClusterDropDown);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.basicOptions.fields.dbClusterDatabaseTypeField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2));
+    I.seeElement(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1));
+    I.click(dbaasPage.tabs.dbClusterTab.optionsCountLocator(2));
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.dbClusterTopologyField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.nodesNumberField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.dbClusterResourceField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.memoryField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.cpuNumberFields);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.advancedOptions.fields.diskSizeInputField);
+    I.seeElement(dbaasPage.tabs.dbClusterTab.createClusterButton);
+    await dbaasAPI.apiUnregisterCluster(clusterName);
+  });
+
+Scenario('PMM-T456 PMM-T490 Verify DB Cluster Steps Background @dbaas @not-pr-pipeline',
+  async (I, dbaasPage, dbaasAPI, adminPage) => {
+    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
+      await dbaasAPI.apiRegisterCluster(process.env.kubeconfig_minikube, clusterName);
+    }
+
+    I.amOnPage(dbaasPage.url);
+    dbaasPage.checkCluster(clusterName, false);
+    I.dontSeeElement(adminPage.fields.timePickerMenu);
+    I.click(dbaasPage.tabs.dbClusterTab.dbClusterTab);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.addDbClusterButton, 30);
+    I.dontSeeElement(adminPage.fields.timePickerMenu);
+    I.click(dbaasPage.tabs.dbClusterTab.addDbClusterButton);
+    I.waitForElement(dbaasPage.tabs.dbClusterTab.monitoringWarningLocator, 30);
+    I.waitForText(dbaasPage.monitoringWarningMessage, 30);
+    await adminPage.verifyBackgroundColor(dbaasPage.tabs.dbClusterTab.optionsCountLocator(1), 'rgb(235, 123, 24)');
```

Comment:
I don't think it is a good idea to check the exact colors here. It should be enough to check that colorAfter some action !== to colorBefore action if this applicable

Reply:
hmm I can do that too, but what is wrong with checking the RGB value? it is still verification of color code which we expect to see, doesn't matter how it looks like on the UI/browser

Reply:
IMHO it does not look like something we should cover with UI tests. I mean assert the exact color.

Reply:
but it is not a visual verification, it is a verification of an expected RGB value, I don't know why it should not be covered, probably when it is changed, we only need to change the expected RGB value here..it is not complex to verify this, and ultimately this is part of user experience while trying to use the app -> UI. 

so look at it like we check for enabled/disabled fields (toggle button etc) in the UI, if we can do that, why we should skip this?

Reply:
just a suggestion. I had a similar test, which started failing because the color changed. So it was causing red job, and some distractions.

Reply:
sure I understand your point

Reply:
Resolving, for now, will keep it like this, probably skip/remove the check-in case it fails a lot due to this change.